SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -78,6 +90,13 @@ bool FSpatialGDKEditorModule::HandleRuntimeSettingsSaved()
 	return true;
 }
 
+bool FSpatialGDKEditorModule::HandleCloudLauncherSettingsSaved()
+{
+	GetMutableDefault<USpatialGDKEditorCloudLauncherSettings>()->SaveConfig();
```

Comment:
Which file is this config sent to in the end?

Reply:
Seems like a file called `SpatialGDKEditorCloudLauncherSettings.ini` is auto-generated in the `.\Game\Saved\Config\Windows` folder. Also updated the setters of the settings so that they're auto-saved when changing them from the pop-up menu as well 👍.

Reply:
Cool:)